assets/blocks/quiz/question-block/question-edit.js

```diff
@@ -42,6 +43,27 @@ import QuestionSettings from './question-settings';
 import { QuestionTypeToolbar } from './question-type-toolbar';
 import SingleQuestion from './single-question';
 
+let questionOptions = Object.entries( types ).map(
```

Comment:
Moved from `question-type-toolbar.js` and sent through props to the `QuestionTypeToolbar` component.